DB/seedDB.js

```diff
@@ -0,0 +1,76 @@
+const faker = require('faker');
+const fs = require('fs');
+
+(function generateData(number, batch) {
+  console.time('timer');
+  const clothes = ['hoodies', 'shirts', 'dress', 'shoes', 'pants', 'jacket'];
+  const randClothesIndex = Math.floor(Math.random() * clothes.length);
+  const keyword = clothes[randClothesIndex];
+  const pictureLength = Math.ceil(Math.random() * 7);
+  let pictures = [];
+  let counter = 0;
+  let data = '';
+
+  for (let j = 0; j < pictureLength; j++) {
+    pictures.push(`https://loremflickr.com/320/240/${keyword}`);
+  }
+
+  for (var i = 0; i < number; i++ ) {
+    let record = `{
+      product: ${i},
+      imageName: ${faker.commerce.productDescription()},
+      color: ${faker.commerce.color().split(' ').join('-')},
+      url:  ${pictures},
+      alt: ${faker.image.abstract()}
+    },`;
+
+    //edge cases of starting and ending values
+    if (i === 0) {
+      record = '[' + record;
+    }
+    if (i === number -1) {
+      let len = record.length;
+      record = record.slice(0, len) + ']';
+    }
+
+    data += record;
+    counter++;
+
+    if (counter % batch === 0) {
+      try {
+        fs.appendFileSync('./DB/primaries.csv', data);
+        console.log('adding: ', i);
+      } catch (err) {
+        console.log('error', i);
+        throw err;
+      }
+      counter = 0;
+      data = '';
+    }
+

```

Comment:
may be worth checking to see if removing "console.log('adding: ', i)" cuts down on your overall time

Reply:
that's a really good point! I will try that out